src/services/email.service.js

```diff
@@ -0,0 +1,42 @@
+import { config } from 'dotenv';
+import sendgrid from '@sendgrid/mail';
+import Debug from 'debug';
+
+config();
+
+const debug = Debug('dev');
+
+const { SENDGRID_API_KEY, SENDGRID_EMAIL, SENDGRID_ACTIVATION_TEMPLATE } = process.env;
+const templates = { accountActivation: SENDGRID_ACTIVATION_TEMPLATE };
+
+
+/**
+     * Initialises the event .
+     * @param {Object} message - The message payload.
+     * @param {String} type - the type of message which corresponds to the mail template
+     * @param {Object | Array} data - can be a list for bulk sending. must contain email parameter
```

Comment:
```suggestion
     * @param {Array} data - can be a list for bulk sending. must contain email parameter
```
Always send it as an array